### PR TITLE
Bryanquigley minor fixups

### PR DESCRIPTION
**Pull request by BryanQuigley**
## Overview

This is a collection of small tweaks found while looking at the memory issues / me breaking new instances from launching

### CODE REVIEW

File: deployment/README.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ To deploy this project's core infrastructure, use the `infra` wrapper script to
 
 ```bash
 $ docker-compose -f docker-compose.ci.yml run --rm terraform
+$ export GIT_COMMIT= (an existing git hash of an image that has been published)
```
[the remark this answers is not in the document]
Comment (BryanQuigley) [reply deployment/README.md:77]: +1, changed

**Comment by BryanQuigley** [comment]
nevermind, that was a real CI failure.. looking into it

File: scripts/infra [hunk 2]
```diff
@@ -16,7 +16,10 @@ Execute Terraform subcommands with remote state management.
 if [[ -n "${GIT_COMMIT}" ]]; then
     GIT_COMMIT="${GIT_COMMIT:0:7}"
 else
-    GIT_COMMIT="$(git rev-parse --short HEAD)"
+    #GIT_COMMIT="$(git rev-parse --short HEAD)"
```
Comment (maurizi) [inline scripts/infra:19]: We generally prefer to not check-in commented-out code without good reason
```suggestion
```

File: scripts/cipublish [hunk 3]
```diff
@@ -32,9 +32,6 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
                 "${DB_AWS_ECR_ENDPOINT}/districtbuilder:${GIT_COMMIT}"
             docker push "${DB_AWS_ECR_ENDPOINT}/districtbuilder:${GIT_COMMIT}"
 
-            docker tag "districtbuilder-manage:${GIT_COMMIT}" \
-                "${DB_AWS_ECR_ENDPOINT}/districtbuilder-manage:${GIT_COMMIT}"
-            docker push "${DB_AWS_ECR_ENDPOINT}/districtbuilder-manage:${GIT_COMMIT}"
```
Comment (maurizi) [inline scripts/cipublish:37]: Have you tried deploying this to staging? I'd expect this change to affect a much larger number of places - I believe this image is what is used in the `StagingAppCLI` / `ProductionAppCLI`, so at the least I'd expect some Terraform changes to remove references to those as well?

If it looks like it's a lot to get into, it's fine by me to just drop this from this PR and open an issue to clean those up in a follow-on task.

**Review by maurizi — COMMENTED** [review]
This mostly looks good to me (I haven't tested the changes).

I left a few comments, but if you've already tested the changes to `cipublish` this is good to go after you address the other comment

File: scripts/cipublish (hunk 3 above)
Comment (BryanQuigley) [reply scripts/cipublish:37]: I added a few related terraform removals (the CLI and setting up the manage ECR).

**Pull request merged**